### PR TITLE
add v1.1 install instructions to vanilla k8s

### PR DESCRIPTION
**Pull request by swiftdiaries**
Release docs for Kubeflow v1.1: https://github.com/kubeflow/website/issues/1984

### CODE REVIEW

File: layouts/shortcodes/config-file-k8s-istio.html
Comment (Jeffwan) [inline layouts/shortcodes/config-file-k8s-istio.html:1]: seems manifest file here point to master. https://github.com/kubeflow/manifests/blob/316c3f16698fce5de8349daf1a59756aeaeec18a/kfdef/kfctl_k8s_istio.yaml#L340

Is that expected?

Comment (swiftdiaries) [reply layouts/shortcodes/config-file-k8s-istio.html:1]: Thanks for the pointer, totally missed that.
Let me make a PR to the manifests repo and get back to you

**Comment by swiftdiaries** [comment]
Pending merging of PR https://github.com/kubeflow/manifests/pull/1484

**Comment by 8bitmp3** [comment]
/assign @rmgogogo  

(Since this "Needs approval from an approver in each of these files: OWNERS")

@Jeffwan @Bobgy If you or someone you know want to be added to OWNERS in `docs/started/k8s/` let me know.

**Comment by Bobgy** [comment]
To move things forward without our approval, I'd suggest setting up OWNERS file for `content/en/docs/started/k8s/` and create a subfolder under `shortcodes` specific to k8s native deployment with an OWNERS file.

**Comment by PatrickXYS** [comment]
Right, I would suggest finding more responsive folks for `content/en/docs/started/k8s/`

File: layouts/shortcodes/config-file-istio-dex.html
Comment (krishnadurai) [inline layouts/shortcodes/config-file-istio-dex.html:1]: @swiftdiaries should this be kfctl_istio_dex.v1.1.0.yaml?

File: content/en/docs/started/k8s/kfctl-k8s-istio.md
Comment (krishnadurai) [inline content/en/docs/started/k8s/kfctl-k8s-istio.md:10]: This is not updated yet.

File: layouts/shortcodes/config-file-istio-dex.html
Comment (swiftdiaries) [reply layouts/shortcodes/config-file-istio-dex.html]: Addressed 🙌🏾

File: content/en/docs/started/k8s/kfctl-k8s-istio.md
Comment (swiftdiaries) [reply content/en/docs/started/k8s/kfctl-k8s-istio.md:10]: I've removed the outdated badge

**Comment by Bobgy** [comment]
Confirming for KFP, did kfctl_istio_dex and kfctl_existing_k8s include KFP 1.0.0 with multi-user separation turned on?

I think our early ideas is KFP multi-user is a key feature to release in Kubeflow 1.1, if a platform doesn't integrate it, it will not be qualified to be called as Kubeflow 1.1.
You may look at integration details in https://github.com/kubeflow/manifests/issues/1364

**Comment by swiftdiaries** [comment]
Hey @Bobgy 
We've included the multi-user KFP in the `kubernetes` stack. [Kustomization for the stack with multi-user KFP](https://github.com/kubeflow/manifests/blob/master/stacks/kubernetes/kustomization.yaml#L23)

**Comment by swiftdiaries** [comment]
There's an [open issue with how it can be used with Dex](https://github.com/kubeflow/kubeflow/issues/4912). I'm trying to give a bare-minimum setup that users can then customize by using the Dex API to get access multi-user KFP. I'm looking into the specific issues and ensure that the OOB KFDef works well soon

**Comment by Bobgy** [comment]
Thank you @swiftdiaries for integrating with k8s_istio manifest!

**Review by krishnadurai — APPROVED** [review]

**Comment by swiftdiaries** [comment]
I'm merging this because it relates to the istio+dex config and the reviewers tagged aren't the right ones for this PR.
I'll follow up with a PR for adding the appropriate OWNERS file

**Pull request merged**
